app/users/authenticator.go

```diff
@@ -29,7 +29,7 @@ func NewAuthenticator(retriever Retriever) *Authenticator {
 // an SDK account ID from Retriever.
 func (a *Authenticator) GetAccountID(r *http.Request) (string, error) {
 	if token, ok := r.Header[TokenHeader]; ok {
```

Comment:
i'm not 100% sure but I think you shouldn't be accessing the headers this way. instead you should use 
r.GetHeader(TokenHeader) which returns a string.

Reply:
There's no GetHeader method on `http.Request` and nothing mentioned in the docs.